pkgs/applications/graphics/inkscape/default.nix

```diff
@@ -91,6 +83,10 @@ stdenv.mkDerivation rec {
       --replace "call('ps2pdf'" "call('${ghostscript}/bin/ps2pdf'"
     patchShebangs share/templates
     patchShebangs man/fix-roff-punct
+
+    # double-conversion is a dependency of 2geom
```

Comment:
I have opened an upstream bug, could you please add a link to the comment? https://gitlab.com/inkscape/inkscape/-/issues/2508